src/components/Designer.vue

```diff
@@ -83,6 +83,20 @@
               color="success"
               @click="submit">submit</v-btn>
           </form>
+          <!-- For font style -->
+          <v-layout>
+            <v-flex
+              xs12
+              sm12
+              md12
+              lg12
+              xl12>
+              <form>
+                <font-style
+                class="my-0 py-0"/>
+              </form>
+            </v-flex>
+          </v-layout>
```

Comment:
Line 86-99 is this needed for this task. I can see this being useful when developing locally, but I don't think that it needs to pushed  to development or master .

Reply:
Moreover, there is a merge conflict that needs to be resolved. I think undoing this file will resolve it

Reply:
LOL I was literally fixing it when you commented 😛 Is it okay to be merged?